sdk/keyvault/azure-security-keyvault-keys/test/ut/CMakeLists.txt

```
@@ -37,10 +38,23 @@ if (MSVC)
   target_compile_options(azure-security-keyvault-keys-test PUBLIC /wd6326 /wd26495 /wd26812)
 endif()
 
-target_link_libraries(azure-security-keyvault-keys-test PRIVATE azure-security-keyvault-keys azure-identity azure-core-test-fw gtest gtest_main gmock)
+target_link_libraries(
+    azure-security-keyvault-keys-test 
+      PRIVATE 
+        azure-security-keyvault-keys 
+        azure-identity azure-core-test-fw  
+        azure-security-attestation
+        gtest 
+        gtest_main 
+        gmock)
 
 # Adding private headers so we can test the private APIs with no relative paths include.
-target_include_directories (azure-security-keyvault-keys-test PRIVATE $<BUILD_INTERFACE:${CMAKE_CURRENT_SOURCE_DIR}/../../src>)
+target_include_directories (
+    azure-security-keyvault-keys-test 
+      PRIVATE 
+        $<BUILD_INTERFACE:${CMAKE_CURRENT_SOURCE_DIR}/../../src>
+        $<BUILD_INTERFACE:${CMAKE_CURRENT_SOURCE_DIR}/../../src/private>
```

Adding src folder should be enough.  You would do `include <private/foo.h> to include headers from src/private, which is easier to read and see as `private` than doing `incude <foo.h>`
```suggestion
        $<BUILD_INTERFACE:${CMAKE_CURRENT_SOURCE_DIR}/../../src>
```

---

done